### PR TITLE
app-toolbar 2.0

### PR DESCRIPTION
**Pull request by blasten**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
     "iron-media-query": "PolymerElements/iron-media-query#^1.0.0",
     "iron-resizable-behavior": "PolymerElements/iron-resizable-behavior#2.0-preview",
     "iron-scroll-target-behavior": "PolymerElements/iron-scroll-target-behavior#^1.0.0",
-    "polymer": "Polymer/polymer#2.0-preview"
+    "polymer": "Polymer/polymer#2.0-preview",
+    "app-route": "polymerelements/app-route#^0.9.0"
```
Comment (frankiefu) [inline bower.json]: Why adding `app-route` here? Only the templates need `app-route` and they have their own bower.json.

Comment (blasten) [inline bower.json]: I might have added it by mistake.

Comment (blasten) [inline bower.json]: fixed

**Comment by frankiefu** [comment]
LGTM

**Pull request merged**